main.tf

```diff
@@ -1,40 +1,47 @@
 module "networking" {
   source = "./modules/networking"
 
-  app-name                = local.app-name
-  id_of_vpc               = module.networking.id_of_vpc
-  both_private_subnets_id = [module.networking.both_private_subnets_id]
+  app_name             = local.app_name
+  availability_zones   = ["us-west-2a", "us-west-2b", "us-west-2c"]
+  cidr                 = "10.0.0.0/16"
+  public_subnets_list  = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+  private_subnets_list = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+  just_count           = 3
+  enable_db_subnets    = true
 }
 
 module "db" {
-  source                     = "modules/db"
-  app-name                   = local.app-name
-  id_of_vpc                  = module.networking.id_of_vpc
+  source                     = "./modules/db"
+  app_name                   = local.app_name
+  availability_zones         = ["us-west-2a", "us-west-2b", "us-west-2c"]
```

Comment:
see the comment about getting `availability_zones` above

Reply:
Done.